### PR TITLE
CHK-4 allow running on dnanexus

### PR DESCRIPTION
**Pull request by gabdank**
modification of checkfiles allowing execution that checks local files and allows execution on DNanexus.
- version of checkfiles was updated to 1.19
- checkfiles will  wait until the portal is indexed (_indexer waiting) prior to execution

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,50 @@ Install required packages for running deploy::
     pyvenv .
     bin/pip install -r requirements-deploy.txt
 
-Deploy
-------
+Deploy to AWS
+-------------
```
Comment (caseylitton) [inline README.md:31]: Add the read me info that Seth put in that other PR.

File: .gitignore [hunk 2]
```diff
@@ -107,3 +107,7 @@ ENV/
 /lib/
 pip-selfcheck.json
 pyvenv.cfg
+
+checkfiles/.DS_Store
```
Comment (caseylitton) [inline .gitignore:111]: change 
```
checkfiles/.DS_Store
```
to
```
.DS_Store
```

**Review by caseylitton — CHANGES_REQUESTED** [review]
I didn't review this very much.  Add the readme from the other PR and remove all the .DS_Store files from the PR.

File: .gitignore (hunk 2 above)
Comment (gabdank) [reply .gitignore:111]: I am not sure about this one, @strattan would know what to add to .gitignore for DNanexus files. Currently, .gitignore supposed to ignore both the repo main directory and the internal checkfiles directory for .DS_Store

File: README.md (hunk 1 above)
Comment (gabdank) [reply README.md:31]: done

**Comment by gabdank** [comment]
@caseylitton I have pushed my changes, not sure about the modification you suggested for .gitignore

**Review by caseylitton — APPROVED** [review]

**Pull request merged**